src/pages/RegistrationForm.tsx

```diff
@@ -70,6 +71,11 @@ function RegistrationForm({ isStudent }: { isStudent: boolean }) {
         loading={loading}
         disabled={loading}
         submitWithoutChange={isRegistering}
+        staticMessage={
+          isStudent
+            ? "⚠️ Students cannot register as mentors"
```

Comment:
Here, I think it would be best to not use Unicode icons as their appearance will depend on the font. Could you add an icon from the `react-icons` package? The one attached below perhaps.
![image](https://github.com/kossiitkgp/KWoC-Frontend/assets/34770591/02f9d688-4cd3-4ce4-ba58-55f4cfcad168)

Reply:
Can you help me. I have got the icon but I am not able to change the color of the icon. I have never used react-icons library.
![image](https://github.com/kossiitkgp/KWoC-Frontend/assets/64408989/ebcb2f92-26b3-41b8-b47f-cffd3bd65cc8)

Code is like this:
![image](https://github.com/kossiitkgp/KWoC-Frontend/assets/64408989/89257671-a079-403c-89be-020160397222)

Reply:
https://stackoverflow.com/questions/56636280/how-to-style-react-icons#56636389

Reply:
Can you send a screenshot of the latest forms?

Reply:
Nothing can be seen If I use the shades on Student Manual and Mentor Manual Button of the home page
![image](https://github.com/kossiitkgp/KWoC-Frontend/assets/64408989/e3931e25-3ff8-488e-b509-1bbd821f552f)

Result of using shades used on submit button 
![image](https://github.com/kossiitkgp/KWoC-Frontend/assets/64408989/36707310-3492-40cf-bf90-537414a8a2e3)

Results while using gray
![image](https://github.com/kossiitkgp/KWoC-Frontend/assets/64408989/db407a69-c06c-4d82-bf6c-08f55d6b81db)

Which one to use?

Reply:
Yes. Thank you!